docs/src/pages/guides/examples.md

```diff
@@ -65,7 +65,7 @@ probes:
           password: somepassword
 ```
 
-Using the configuration above, Monika will send a POST request to http://www.foo.com/login.php with the defined request's body.
+Using the configuration above, Monika will send a POST request to <http://www.foo.com/login.php> with the defined request's body.
```

Comment:
maybe better use the inline code backtick? `http://www.foo.com/login.php`.

Reply:
done. updated.